gui/mainwindow.py

```
@@ -288,7 +288,10 @@ def goto_new_patient(self):
         self.show_startup()
 
     def goto_resume_patient(self):
-        # TODO : implement start from current_settings.json
+        settings_file = SettingsFile(self.config["settings_file_path"])
+        settings = settings_file.load()
+        self.settings.update_config(settings)
```

If the file doesn't exist (or corrupted), the dict returned is empty, in this case we should not update the settings. Maybe we should raise an error window saying that we cannot go back to previous patient.

---

you are perfectly right! Thanks! the same applies after a software upgrade that could introduce a new parameter.

---

solved in commit 5b7bedb